hack/make-rules/test-e2e-node.sh

```diff
@@ -56,7 +56,7 @@ runtime_config=${RUNTIME_CONFIG:-}
 ssh_user=${SSH_USER:-"${USER}"}
 ssh_key=${SSH_KEY:-}
 ssh_options=${SSH_OPTIONS:-}
-kubelet_config_file=${KUBELET_CONFIG_FILE:-"test/e2e_node/jenkins/default-kubelet-config.yaml"}
+kubelet_config_file=${KUBELET_CONFIG_FILE:-"test/e2e_node/infraconfig/default-kubelet-config.yaml"}
```

Comment:
As you're changing hack/ can you also do the same with `hack/jenkins` as you do with test/e2e_node/jenkins ?